### DISCUSSION TITLE
Table of planned and implemented controls?

### DISCUSSION THREAD

**Discussion by iot-crazy** [dotnet/maui#7547] [Q&A]
I'm certain that I recently saw a table showing the planned MAUI controls and which ones were implemented, but now I cannot find it.
Can anybody point me towards this table if it still exists?

**Comment by john-hollander** [comment]
It was here:

https://github.com/dotnet/maui/wiki/status

But the controls mentioned are complete, and they are in the issue resolution stage now.

This is the last edit before they wiped the page:
https://github.com/dotnet/maui/wiki/Status/48a215bc5407843201264d24c7746a7364e30dc4

**Reply by TonyK-FmGroup** [reply]
Thank you, I'm trying to find out how to mark your reply as the answer.
